src/main/java/htsjdk/variant/vcf/VCFHeader.java

```diff
@@ -68,7 +68,7 @@
     private final Map<String, VCFFormatHeaderLine> mFormatMetaData = new LinkedHashMap<String, VCFFormatHeaderLine>();
     private final Map<String, VCFFilterHeaderLine> mFilterMetaData = new LinkedHashMap<String, VCFFilterHeaderLine>();
     private final Map<String, VCFHeaderLine> mOtherMetaData = new LinkedHashMap<String, VCFHeaderLine>();
-    private final List<VCFContigHeaderLine> contigMetaData = new ArrayList<VCFContigHeaderLine>();
+    private final Map<String, VCFContigHeaderLine> contigMetaData = new LinkedHashMap<>();
```

Comment:
I would also disagree about this, for the same reason; the ordering is a data storage requirement but not an API restriction. If in the future we switched to a different implementation of an ordered map then we'd need to update our code unnecessarily.